src/apiKey.ts

```diff
@@ -0,0 +1,3 @@
+export const keyWeather = '351d2714b99a8b9522929c7b8ea79b2f';
+export const keyСurrency = '5bca3cf01e974582b14f601e7af836ca';
+export const keyMap = 'fc7c12bf-a279-45f1-b88e-a8381d09b00a';
```

Comment:
if possible, make sure that your vscode redactor or whatever else puts a new line at the end of the file.

Reply:
looks like secret keys, maybe put it in `process.env`?